src/app/services/nearbyCities.service.js

```diff
@@ -0,0 +1,52 @@
+export default class NearbyCitiesService {
+    constructor($http) {
+        this.$http = $http;
+    }
+
+    getNearbyCities(city) {
+        let lat = 0;
+        let lon = 0;
```

Comment:
Why you declare here these variables?? are used only inside the success response